basis_set_exchange/manip.py

```diff
@@ -381,6 +452,13 @@ def make_general(basis, skip_spdf=False, use_copy=True):
 
         el['electron_shells'] = newshells
 
+    # If the basis was read in from a segmented format, it will have
+    # duplicate primitives, and so a pruning is necessary
+    prune_basis(basis, False)
+    # We also have to prune the contractions, since they might be
+    # split across shells
+    prune_general_contractions(basis, False)
```

Comment:
Is this ever true? I thought at this point we only have one shell per AM, so it should never be split across shells

Reply:
Yes. See #166